Api/V8/Service/ModuleService.php

```diff
@@ -155,8 +223,12 @@ public function getRecords(GetModulesParams $params, Request $request)
         if ($data && $limit !== BeanManager::DEFAULT_LIMIT) {
             $totalPages = ceil($realRowCount / $size);
 
-            $paginationMeta = $this->paginationHelper->getPaginationMeta($totalPages, count($data));
-            $paginationLinks = $this->paginationHelper->getPaginationLinks($request, $totalPages, $number);
+            $paginationMeta
+                = $this->paginationHelper->getPaginationMeta($totalPages,
+                count($data));
+            $paginationLinks
+                = $this->paginationHelper->getPaginationLinks($request,
+                $totalPages, $number);
```

Comment:
These method argument formats don't match [PSR](https://www.php-fig.org/psr/psr-2/#44-method-arguments)

_Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line._